Plots/Colors/CB_Height.py

```diff
@@ -64,68 +64,67 @@
 
 fig = plt.figure(figsize=(12, 12))
 
-
-def Plot(color, row, col, pos, title):
-
-    # Generate axes, using Cartopy, drawing coastlines, and adding features
+def Plot(color,row, col, pos, title):   
+# Generate axes, using Cartopy, drawing coastlines, and adding features
     projection = ccrs.PlateCarree()
     ax1 = plt.subplot(row, col, pos, projection=projection)
     ax1.coastlines(linewidths=0.5)
     ax1.add_feature(cfeature.LAND, facecolor="lightgray")
-
+    
     # Import an NCL colormap
-    newcmp = color  #gvcmaps.BlAqGrYeOrRe
-
+    newcmp = color #gvcmaps.BlAqGrYeOrRe
+    
     # Contourf-plot data
-    t.plot.contourf(ax=ax1,
-                    transform=projection,
-                    levels=40,
-                    vmin=100,
-                    vmax=1600,
-                    cmap=newcmp,
-                    cbar_kwargs={
-                        "orientation": "vertical",
-                        "ticks": np.arange(100, 1600, 100),
-                        "label": "",
-                        "shrink": 0.8
-                    })
-
+    hgt = t.plot.contourf(
+        ax=ax1,
+        transform=projection,
+        levels=40,
+        vmin=100,
+        vmax=1600,
+        cmap=newcmp,
+        add_colorbar=False)
+
+    # Add color bar
+    cbar_ticks = np.arange(100, 1600, 100)
+    cbar = plt.colorbar(hgt, orientation='vertical', shrink=0.8, pad=0.05, extendrect=True)
+    cbar.ax.tick_params(labelsize=10)
+    cbar.set_ticks(cbar_ticks)
```

Comment:
The colorbar ticks can be set with `plt.colorbar()` by using the `ticks` argument. What you have works, but this way is more concise if you would like to use it instead.